test/scratch.html

```diff
@@ -217,6 +217,13 @@
 import * as Inputs from "@observablehq/inputs";
 
 document.body.appendChild(Inputs.range());
+document.body.appendChild(Inputs.text({
+    type: "date",
+    label: "2020 Election Campaign Date",
+    min: "2020-01-01",
+    max: "2020-11-04",
+    value: "2020-07-07"
+    }));
 
   </script>
-</body>
+</body>
```

Comment:
```suggestion
</body>

```

Reply:
(I tried to revert this line change, not sure I got it right)

Reply:
Looks good to me